### DISCUSSION TITLE
Set alignment of horizontal buttons stack

### DISCUSSION THREAD

**Discussion by Laho812** [Clooos/Bubble-Card#367] [Questions about config, custom styles and templates] [answered]
Hi all,
I want the horizontal buttons stack to be right aligned. 
Is there a way to change the alignment e.g. using styles already?

Thanks in advance!

**Comment by MrBearPresident** [comment]
Did you solve this?
Can this be closed?

**Comment by Laho812** [comment, marked as answer]
Yes, indeed I solved it. I wanted to achieve a single "back" button in the lower right corner for mobile devices.

```
.bubble-horizontal-buttons-stack-card-container {
    margin: auto 20px auto auto;
} 
```

![Screenshot_20241229_214320_Home Assistant.jpg](https://github.com/user-attachments/assets/b21852d4-415c-4a4b-bc0e-d5a7a0d00cb4)
